pkg/registry/cratesio/rust.go

```diff
@@ -11,16 +11,17 @@ import (
 )
 
 type release struct {
-	Version string
-	Date    time.Time
+	Version   string
+	Date      time.Time
+	MUSLBuild bool
 }
 
-func newRelease(version, dateStr string) release {
+func newRelease(version, dateStr string, muslBuild bool) release {
```

Comment:
`bool` in method parameters is a bit odd. Would it make sense to have both
- `newRelease(version,data)`
- `newReleseWithMUSL(version,data)`
which both populate the release struct differently?

Reply:
meh it's generated by the command referenced in the comment below so I figure it's not necessarily meant for actual use, just codegen ease